pkg/vulnsrc/ubuntu/ubuntu.go

```diff
@@ -170,7 +170,8 @@ func defaultPut(dbc db.Operation, tx *bolt.Tx, advisory interface{}) error {
 			}
 
 			adv := types.Advisory{}
-			if status.Status == "released" {
+			adv.Status = StatusFromUbuntuStatus(status.Status)
+			if adv.Status == types.StatusFixed {
 				adv.FixedVersion = status.Note
 			}
```

Comment:
We don't want to store the "fixed" status in DB so that we can keep the DB size as small as possible.

```suggestion
			status = StatusFromUbuntuStatus(status.Status)
			if status == types.StatusFixed {
				adv.FixedVersion = status.Note
			} else {
				// Store the status only if it's unfixed
				adv.Status = status
			}
```

Reply:
Ah, this makes a lot of sense. Thanks for the suggested change @knqyf263! We already have code on our side which treats advisories with `FixedVersion` set as status fixed (we use this for Red Hat advisories) so this approach will be consistent too.

Reply:
Thanks for understanding. Since this DB is frequently downloaded, the size is essential for us.